### PR TITLE
feat(storybook):  adding storybook as a dependency

### PR DESCRIPTION
**Pull request by carolinasamp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,22 +19,33 @@
...
     "@types/react": "^16.9.52",
     "@types/react-dom": "^16.9.8",
+    "babel-loader": "^8.0.6",
+    "clean-webpack-plugin": "^3.0.0",
+    "html-webpack-plugin": "^3.2.0",
     "jest": "^26.5.3",
     "jest-css-modules-transform": "^4.0.1",
-    "ts-jest": "^26.4.1"
+    "ts-jest": "^26.4.1",
+    "react-is": "^16.13.1",
```
Comment (arturbani) [inline package.json:48]: Are we using this dependency somewhere?

File: package.json [hunk 2]
```diff
@@ -18,22 +19,33 @@
...
+    "@storybook/react": "^6.0.26",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@types/jest": "^26.0.14",
     "@types/testing-library__react": "^10.2.0",
     "@types/react": "^16.9.52",
     "@types/react-dom": "^16.9.8",
+    "babel-loader": "^8.0.6",
+    "clean-webpack-plugin": "^3.0.0",
+    "html-webpack-plugin": "^3.2.0",
```
Comment (arturbani) [inline package.json:44]: We can remove those dependencies since we removed webpack!

File: package.json [hunk 3]
```diff
@@ -18,22 +19,33 @@
...
     "jest": "^26.5.3",
     "jest-css-modules-transform": "^4.0.1",
-    "ts-jest": "^26.4.1"
+    "ts-jest": "^26.4.1",
+    "react-is": "^16.13.1",
+    "ts-loader": "^6.2.0"
```
Comment (arturbani) [inline package.json:49]: We can also remove this dependency

File: select/selectSearchable.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import SelectSearchable from "./selectSearchable";
+import { items } from "../app";
```
Comment (gabecar) [inline select/selectSearchable.stories.tsx:4]: this import is wrong, the items mock is in "../test/mocks/items"

**Review by gabecar — CHANGES_REQUESTED** [review]

File: select/selectSearchable.stories.tsx (hunk 4 above)
Comment (carolinasamp) [reply select/selectSearchable.stories.tsx:4]: yes, i'm adjusting

**Comment by carolinasamp** [comment]
♻️

**Review by gabecar — APPROVED** [review]

**Pull request merged**